### PR TITLE
hotfix: add package needed for building docker image on apple silicon

### PR DESCRIPTION
**Pull request by nathanfranklin**
## Overview: ##

add package needed for apple silicon docker build

## Related Jira tickets: ##

None

## Notes ##

~Nathan June 8th:  hmm, this only works with emulation (i.e. using  `export DOCKER_DEFAULT_PLATFORM=linux/amd64` ) looking at https://pythonspeed.com/articles/docker-build-problems-mac/.   Actually, still not working.  images built and pulled but then starting containers get stuck. no errors.  just spinning.~

Nathan Jul 28th: working now

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ ENV POETRY_HOME=/opt/poetry
 ENV PATH="$POETRY_HOME/bin:$PATH"
 ENV PYTHONPATH "${PYTHONPATH}:/app"
 
-RUN apt-get update && apt-get install -y curl
+RUN apt-get update && apt-get install -y curl gdal-bin libgdal-dev gcc g++
```
Comment (taoteg) [inline Dockerfile:10]: Will this have on impact on a non-apple system, like unused packages or throwing build errors?

**Review by taoteg — APPROVED** [review]
Assuming non non-Apple sideffects, looks fine.

File: Dockerfile (hunk 1 above)
Comment (nathanfranklin) [reply Dockerfile:10]: was only throwing errors on building on apple silicon (new m2 laptop).

but I put the this PR in draft as it still doesn't run.  now the containers are built but it gets stuck on "Creating container". i gotta do some digging to figure that out.  so for right now, I can't run protx-dashboard locally on m2.

Comment (nathanfranklin) [reply Dockerfile:10]: @taoteg @sophia-massie ,  yeah, i'm not certain. can you confirm that it works on your end?

**Review by sophia-massie — APPROVED** [review]

**Pull request merged**